### PR TITLE
Maid 1027 test account sentinel

### PR DESCRIPTION
**Pull request by inetic**
https://maidsafe.atlassian.net/browse/MAID-1027

[Review on Reviewable](https://reviewable.io/reviews/maidsafe/sentinel/25)

### CODE REVIEW

**Comment by maidsafe-highfive** [comment]
r? @benjaminbollen

(maidsafe_highfive has picked a reviewer for you, use r? to override)

File: src/account_sentinel.rs [hunk 1]
```diff
@@ -66,6 +66,72 @@ impl<Request, Name, Claim> AccountSentinel<Request, Name, Claim>
         let mut claims = map.iter().map(|(_, ref claim)| claim.clone())
                             .collect::<Vec<_>>();
         claims.sort();
-        Some(claims[claims.len() / 2].clone())
+        Some(claims[(claims.len() - 1) / 2].clone())
```
Comment (benjaminbollen) [inline src/account_sentinel.rs:69]: Are you sure we need to introduce -1 here?  Should it not instead require a correction in the test?

Comment (inetic) [inline src/account_sentinel.rs:69]: No, this is on purpose:

| sequence | previous calculation | new calculation |
| --- | --- | --- |
| [0] | 0 | 0 |
| [0,1] | 1 | 0 |
| [0,1,2] | 1 | 1 |
| [0,1,2,3] | 2 | 1 |
| [0,1,2,3,4] | 2 | 2 |

Comment (benjaminbollen) [inline src/account_sentinel.rs:69]: No after quick check, seems correct behaviour we intended. Nice!

**Pull request merged**